Automattic/wp-calypso
# Signup: add `designFirstSignup` AB test

This adds the test stub for the themes-first Signup test. The test is set to the future with a 0/100 split. The test will split traffic from the default `onboarding` Signup once enabled.

Epic: Automattic/zelda-private#119
Requires: Automattic/wp-calypso#36825

**To Test:**
- visit `/start/`
- use the AB tool to select the `designFirst` variant of the `designFirstSignup` test
- on reload, you should see the theme step of the `design-first` Signup flow
